src/Entity/Group.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Exception\GroupAlreadyApprovedException;
+use Doctrine\ORM\Mapping as ORM;
+use libphonenumber\PhoneNumber;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * This entity represents a group.
+ *
+ * @ORM\Table(
+ *   name="groups",
+ *   uniqueConstraints={
+ *     @ORM\UniqueConstraint(name="group_uuid_unique", columns="uuid"),
+ *     @ORM\UniqueConstraint(name="group_canonical_name_unique", columns="canonical_name"),
+ *     @ORM\UniqueConstraint(name="group_slug_unique", columns="slug")
+ *   },
+ *   indexes={
+ *     @ORM\Index(name="group_status_idx", columns="status")
+ *   }
+ * )
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\GroupRepository")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Group extends BaseGroup
+{
+    public function __construct(
```

Comment:
Constructor should live in the base class.

Reply:
Added.